app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repetition and improve test readability. This will help adhere to the 'Don't Repeat Yourself' principle from the checklist.